backend/controllers/allocateKeyWorker.js

```diff
@@ -80,13 +74,10 @@ module.exports = ({ allocationService, keyworkerApi, oauthApi }) => {
             isManualAllocation &&
             `${offender.keyworkerDisplay} ${formatNumberAllocated(offender.numberAllocated)}`,
           keyworkerStaffId: staffId,
-          keyworkerList: sortedSelectableKeyworkers.map((keyworker) => {
```

Comment:
hmm - are there no tests for this page? Maybe would be good to add ones?

Reply:
There are tests from the previous PR. This change is none functional.